pyOlog/OlogClient.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, url=None, username=None, password=None, ask=True):
         self.verify = False
         username = _conf.get_username(username)
         password = _conf.get_value('password', password)
+        self._old_olog_api = _conf.get_value('old_olog_api', old_olog_api)
```

Comment:
Will it handle the case when the value is not set in the config file?

Reply:
Yep, there is some logic that does that.
I updated and tested this code more.